test/e2e/e2e.go

```diff
@@ -491,7 +512,7 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 		podChan, errChan := make(chan *v1.Pod), make(chan error)
 		go func() {
 			defer ginkgo.GinkgoRecover()
-			checkContinuousConnectivity(f, "", "connectivity-test-continuous", "8.8.8.8", 53, 30, podChan, errChan)
+			checkContinuousConnectivity(f, "", "connectivity-test-continuous", "8.8.8.8", 53, 30, 30, podChan, errChan)
```

Comment:
should be 10 tries? or is there a reason to make it 30?

Reply:
because it can take more than 20 seconds for the delete to happen, so we were reliably not actually testing connectivity during the actual pod deletion.

Reply:
We really need a connectivity-checker test that doesn't have a hard-coded timeout, and instead runs until told to quit.